### PR TITLE
Use tb64senc for base64Decode on aarch64

### PR DESCRIPTION
**Pull request by vdimir**
Changelog category (leave one):

- Not for changelog (changelog entry is not required)

Ref #33716

### CODE REVIEW

**Comment by vdimir** [comment]
It's temporary solution, I will replace used library https://github.com/ClickHouse/ClickHouse/issues/31643

**Review by alexey-milovidov — APPROVED** [review]
LGTM

File: src/Functions/FunctionBase64Conversion.h [hunk 1]
```diff
@@ -124,13 +124,26 @@ class FunctionBase64Conversion : public IFunction
 
             if constexpr (std::is_same_v<Func, Base64Encode>)
             {
+                /*
+                 * Some bug in sse arm64 implementation?
+                 * `base64Encode(repeat('a', 46))` returns wrong padding character
+                 */
+#if (__aarch64__)
```
Comment (azat) [inline src/Functions/FunctionBase64Conversion.h:131]: `#if defined(__aarch64__)`

File: src/Functions/FunctionBase64Conversion.h [hunk 2]
```diff
@@ -124,13 +124,26 @@ class FunctionBase64Conversion : public IFunction
...
+#else
                 outlen = _tb64e(reinterpret_cast<const uint8_t *>(source), srclen, reinterpret_cast<uint8_t *>(dst_pos));
+#endif
             }
             else if constexpr (std::is_same_v<Func, Base64Decode>)
             {
                 if (srclen > 0)
                 {
-                    outlen = _tb64d(reinterpret_cast<const uint8_t *>(source), srclen, reinterpret_cast<uint8_t *>(dst_pos));
+#if (__aarch64__)
```
Comment (azat) [inline src/Functions/FunctionBase64Conversion.h:141]: `#if defined(__aarch64__)`

**Pull request merged**